expressEngine/user/service/user.js

```diff
@@ -14,40 +14,43 @@ class Service extends BaseService {
   }
 
   registerUser = async ({ email, password, ...rest }) => {
-    const isExist = await DBService.count(User, { email });
-    if (isExist) {
-      throw error.alreadyRegistered;
+    try {
+      const isExist = await DBService.count(User, { email });
+      if (isExist) {
+        throw new Error(error.alreadyRegistered);
```

Comment:
There is no need to throw an object of Error, you can directly throw _error.alreadyRegistered_